lib/ProductOpener/Images.pm

```diff
@@ -219,7 +219,12 @@ sub scan_code($) {
 	my $magick = Image::Magick->new();
 	my $x = $magick->Read($file);
 	local $log->context->{file} = $file;
-	if ("$x") {
+	
+	# Disabling the check for $x, as ImageMagick triggers an exception for some images
+	# that it can read anyway
+	# Exception 365: CorruptImageProfile `xmp' @ warning/profile.c/SetImageProfileInternal/1704
+	# if ("$x") {
+	if (0) {
```

Comment:
You could check for a number above 400, instead of any response.
https://www.imagemagick.org/script/perl-magick.php#exceptions says " A status code less than 400 is a warning. This means that the operation did not complete but was recoverable to some degree. A numeric code greater or equal to 400 is an error and indicates the operation failed completely."

Reply:
I didn't know that, thank you. I'll change the code accordingly.